package.json

```diff
@@ -90,7 +90,7 @@
   },
   "engines": {
     "node": "8",
-    "npm": "5"
+    "npm": "6"
```

Comment:
wil je npm 6? npm 5 is LTS en meer stable.. ik heb soms issues met 6 en ken nog geen reden om naar 6 te switchen

Reply:
Ait, ik kreeg een foutmelding dus daarom even naar 6 gezet, zal het even terug zetten en weer testen